client/post-editor/post-editor.jsx

```diff
@@ -1178,6 +1180,7 @@ const enhance = flow(
 				previewUrl: getEditorPostPreviewUrl( state ),
 				isAutosaving: isEditorAutosaving( state ),
 				isLoading: isEditorLoading( state ),
+				isLoaded: isEditorLoaded( state ),
```

Comment:
I think you don't need to modify this particular file at all. The `isLoaded` prop is not used anywhere in the connected component. And this component is the UI for the Classic (TinyMCE) editor. And I assume you need to setup spellchecker only for the Gutenberg iframe? Or does Classic/TinyMCE also need fixing?

Reply:
Thanks for pointing this out. We _do_ in fact want an event to fire when the the classic editor loads as well. In that case, the implementation in this file is incomplete - what would be the appropriate location to call a similar "loaded" event for TinyMCE?

Not sure if it's reasonable to re-use the same `EDITOR_LOADED` event since Desktop doesn't need to distinguish between the two. If it's more appropriate, we could consider differentiating the two - something like `GUTENBERG_LOADED` vs `TINYCME_LOADED`?

Reply:
TinyMCE also loads the edited content into an `iframe`, so if you want to attach the spellchecker to the `input` event stream in that `iframe`, you should trigger an `EDITOR_LOADED` action, too.

The best way to do that is in the `<TinyMCE onInit>` callback, i.e., in the `PostEditor.onEditorInitialized` callback.

But I admit I don't fully understand the changes done in https://github.com/Automattic/wp-desktop/pull/749.

You call `spellCheckHandler.attachToInput( dummy$ )`, with an empty stream of `input` events. I suppose that input stream should be at some point in time replaced with the real stream coming from `document.body`? But I don't see where that happens.

The `editor-loaded` event is fired at a time when the `iframe` is loaded and its `contentDocument` can be used. Why create a dummy stream then, when we can create a real one from the iframe's document?

The `electron-spellchecker` [creates the `input` stream from the `document.body` element](https://github.com/electron-userland/electron-spellchecker/blob/7227d61415c47dfa247828da3b62bbb3330095aa/src/spell-check-handler.js#L217), but which document is that? The top-level frame, or the content iframe? I think it should be the content iframe of the Gutenframe or TinyMCE. But the `electron-spellchecker` code runs in the top-level frame -- is it capturing the input from the right frame then?

Reply:
> something like `GUTENBERG_LOADED` vs `TINYCME_LOADED`?

At this moment, there seems to be no difference between how these two events would be handled. Therefore, a single `EDITOR_LOADED` looks fine to me 🙂

Reply:
Refactored in 898ca2f and a15dc0d